### PR TITLE
Fix for NMB-27865: DSL evaluation issue using generateDsl format

### PR DESCRIPTION
**Pull request by urvashisingh**
Creating PR to pull the changes from https://github.com/electric-cloud/EC-DslDeploy/pull/10.

### CODE REVIEW

**Comment by urvashisingh** [comment]
Creating the PR was the smoothest route to merge my changes from the original location. But now I am hitting this requirement of needing 2 approving reviews. So @lrochette could you be one of the reviewers? Note that I *will* revert the README changes that got pulled into the PR.

**Comment by urvashisingh** [comment]
@imago-storm could you review this change as well? thanks

File: README.md [hunk 1]
```diff
@@ -1,51 +1,3 @@
 # EC-DslDeploy
-Deployment of your DSL code structure to your server.
```
Comment (imago-storm) [inline README.md]: I guess this README should remain the the old repository only. And probably it is better to place the movement note into the repository description.

Comment (lrochette) [reply README.md]: Great idea. I will do that

**Pull request merged**